src/data/content/design-system/fondamenti/bordi-e-raggi.yaml

```diff
@@ -0,0 +1,212 @@
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer, Qui per curiosità #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Progettazione interfaccia, Sviluppo interfaccia]
+  
+  sectionsEditorial:
+    - full: true
+      noSpace: false
+      paddingLeft: true
+      title: null
+      text: null
+      background: null
+      components:
+        - name: TextImageCta
+          title: A cosa servono
+          headingLevel: 2
+          text: |
+            ### Bordi
+            I bordi sono linee che delimitano i **contorni di elementi** oppure oggetti presenti in un'interfaccia. 
+            
+            Possono essere applicati a componenti come pulsanti, campi di input, sfondi, immagini o qualsiasi area definita. 
+            
+            I bordi contribuiscono a dare enfasi alla forma e alla struttura di un elemento, possono variare in termini di spessore e stile e creano una separazione fra l'elemento e ciò che lo circonda.
+
+            ### Raggi
+            I raggi definiscono il grado di **curvatura degli angoli** di un elemento che ha un bordo. La misura di un raggio determina quanto gli angoli dell'elemento sono arrotondati.
+
+            Ad esempio, se un raggio è impostato su `4px`, gli angoli dell'elemento saranno arrotondati con una curvatura di questa entità.
+
+            Sono anche conosciuti con il termine tecnico di "border-radius".
+        
+        - name: TextImageCta
+          title: Come usarli
+          headingLevel: 2
+          text: |
+            ### Bordi arrotondati
+            L'uso di bordi arrotondati conferisce leggerezza visiva agli elementi di un'interfaccia. Per questo motivo sono raccomandati per pulsanti, campi di input, card e simili.
+
+            Per dare più enfasi a elementi chiave dell'interfaccia, puoi aumentare la dimensione del raggio dei bordi che vorresti mettere in risalto.
+
+            ### Bordi squadrati
+
+            I bordi squadrati si addicono maggiormente a elementi che richiedono una presentazione più formale e strutturata, come modali o riquadri informativi.
```

Comment:
Se bordo è il concetto della linea di bordo presente o meno e in che peso. 
Allora per i raggi io userei il concetto di angoli, curvi o squadrati. 
 
Perché ci possono essere elementi senza bordi (senza linea di bordo intendo) con angoli squadrati o più o meno curvi.